sdk/mixedreality/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/aio/_remote_rendering_client_async.py

```diff
@@ -0,0 +1,339 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import asyncio
+import base64
+import pickle
+from functools import partial
+from typing import TYPE_CHECKING
+
+from azure.core.async_paging import AsyncItemPaged
+from azure.core.credentials import AccessToken, AzureKeyCredential
+from azure.core.pipeline.policies import AsyncBearerTokenCredentialPolicy
+from azure.core.polling import AsyncLROPoller, AsyncPollingMethod
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.mixedreality.authentication._shared.authentication_endpoint import \
+    construct_endpoint_url
+from azure.mixedreality.authentication._shared.aio.mixed_reality_token_credential import \
+    get_mixedreality_credential
+from azure.mixedreality.authentication._shared.aio.mixedreality_account_key_credential import \
+    MixedRealityAccountKeyCredential
+from azure.mixedreality.authentication._shared.aio.static_access_token_credential import \
+    StaticAccessTokenCredential
+
+from .._generated.aio import RemoteRenderingRestClient
+from .._generated.models import (Conversion,
+                                 ConversionSettings, ConversionStatus,
+                                 CreateConversionSettings,
+                                 CreateSessionSettings, SessionProperties,
+                                 SessionSize, SessionStatus,
+                                 UpdateSessionSettings)
+from .._version import SDK_MONIKER
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Callable, Union
+
+    from azure.core.credentials import TokenCredential
```

Comment:
this should use the async version.
```suggestion
    from azure.core.credentials_async import AsyncTokenCredential
```